src/types.jl

```diff
@@ -35,19 +35,20 @@ function get_registry(
 )
     if haskey(cache.registries, registry_url)
         registry_path = path(cache, registry_url)
+        default_branch = splitpath(String(readchomp(`git symbolic-ref refs/remotes/origin/HEAD`)))[end]
 
         if !ispath(registry_path)
             mkpath(path(cache))
-            run(`git clone $registry_url $registry_path --branch=master`)
+            run(`git clone $registry_url $registry_path --branch=$default_branch`)
```

Comment:
Here I think we can just remove the `branch` option as the clone should default to the default branch

Reply:
Seems reasonable to me.